frontend/src/components/MapView/LeftPanel/ChartsPanel/index.tsx

```diff
@@ -275,6 +275,8 @@ const ChartsPanel = memo(
       [string, string]
     >(['', '']);
     const [showSlider, setShowSlider] = useState(true);
+    const [maxChartValue, setMaxChartValue] = useState<number[]>([]);
```

Comment:
let's pluralize this variable names to make it clear that there are lists?

Reply:
or maybe these could be of type `number` instead and the setters handle the logic to keep the larger / smaller option, wdyt?